packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/datebox.mask.tests.js

```diff
@@ -1004,6 +1004,33 @@ module('Events', setupModule, () => {
 
         assert.strictEqual(event.isDefaultPrevented(), true, `the ${DROP_EVENT_NAME} event is prevented`);
     });
+
+    QUnit.test('change year via scroll should log proper year in on value change event (T1229926)', function(assert) {
```

Comment:
Your test is always green. Before calling for a review, ensure that the test reproduces the scenario.

Reply:
[889bce5](https://github.com/DevExpress/DevExtreme/pull/27418/commits/889bce591bb90b93070b93c7ab94ad1047175923)